pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -493,14 +494,10 @@ cdef class _Timestamp(ABCTimestamp):
 
     @property
     def _time_repr(self) -> str:
-        result = f'{self.hour:02d}:{self.minute:02d}:{self.second:02d}'
-
-        if self.nanosecond != 0:
-            result += f'.{self.nanosecond + 1000 * self.microsecond:09d}'
-        elif self.microsecond != 0:
-            result += f'.{self.microsecond:06d}'
-
-        return result
+        fmt = '%H:%M:%S'
```

Comment:
does this change any of the benchmarks?

Reply:
Looks like we suffer a penalty for the additional super() function call, and then there's an additional performance impact when we actually need to process nanoseconds. 

```
       before           after         ratio
     [6efb0b20]       [0c4ef36f]
     <master>         <strftime>
+      3.61±0.1μs       4.40±0.2μs     1.22  tslibs.timestamp.TimestampStrftimeMethod.time_strftime('%Y-%m-%d %H:%M:%S.%f')
+      3.29±0.1μs      3.68±0.05μs     1.12  tslibs.timestamp.TimestampStrftimeMethod.time_strftime('%Y-%m-%d %H:%M:%S')
+      8.91±0.2μs       9.90±0.4μs     1.11  tslibs.timestamp.TimestampProperties.time_month_name(None, None)
-        409±20ns         308±10ns     0.75  tslibs.timestamp.TimestampProperties.time_days_in_month(<UTC>, 'B')
```